cloud_connector.go

```diff
@@ -0,0 +1,71 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/goccy/go-json"
+)
+
+type CloudConnectorRulesResponse struct {
+	Response
+	Result []CloudConnectorRule `json:"result"`
+}
+
+type CloudConnectorRuleParameters struct {
+	Host string `json:"host"`
+}
+
+type CloudConnectorRule struct {
+	ID          string                       `json:"id"`
+	Enabled     bool                         `json:"enabled"`
```

Comment:
```suggestion
	Enabled     *bool                         `json:"enabled"`
```

https://github.com/cloudflare/cloudflare-go/blob/master/docs/conventions.md#booleans

Reply:
Fixed